policyengine_us/variables/gov/irs/income/taxable_income/adjusted_gross_income/irs_gross_income/social_security/tax_unit_magi_excess.py

```diff
@@ -0,0 +1,28 @@
+from policyengine_us.model_api import *
+
+
+class tax_unit_magi_excess(Variable):
```

Comment:
This is more appropriate right?
```suggestion
class tax_unit_ss_combined_income_excess(Variable):
```